components/log-viewer-webui/server/src/app.ts

```diff
@@ -56,6 +57,11 @@ const app = async ({
         await server.register(S3Manager, {region: settings.StreamFilesS3Region});
     }
 
+    await server.register(MongoReplicaServerPlugin, {
+        mongoUri: "mongodb://localhost:27017",
+        dbName: "clp-query-results",
```

Comment:
1. For consistency with the Fastify-mongodb plugin, let's rename `dbName` as `database.`; similarly, let's break the `mongoUri` property into `host` and `name`.
2. We don't want to hardcoded the values here. They should come from `settings.xxx`. Note we need to add another entry in the settings.json file like `MongoDbQueryResultsCollectionName`.